packages/kbn-bazel-packages/BUILD.bazel

```diff
@@ -58,6 +58,7 @@ TYPES_DEPS = [
   "//packages/kbn-utils:npm_module_types",
   "//packages/kbn-std:npm_module_types",
   "//packages/kbn-synthetic-package-map:npm_module_types",
+  "//packages/kbn-kibana-manifest-parser:npm_module_types",
```

Comment:
Was this supposed to be part of this PR?

Reply:
Nope! Thank you